packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -729,3 +730,67 @@ test(
   'metadataObjectFromCMRFile throws PreconditionFailed when ETag does not match CMR UMMG JSON metadata file',
   testMetadataObjectFromCMRFile('s3://bucket/fake.cmr.json')
 );
+
+test.serial('publish2CMR passes cmrRevisionId to publishECHO10XML2CMR', async (t) => {
+  const cmrFileObject = { filename: 'test.cmr.xml', granuleId: 'testGranuleId' };
+  const updatedXmlFile = { ...cmrFileObject, metadataObject: {} };
+
+  const publishECHO10XML2CMRSpy = sinon.spy(() => Promise.resolve());
+  const revertPublishECHO10XML2CMRSpy = cmrUtil.__set__('publishECHO10XML2CMR', publishECHO10XML2CMRSpy);
+  const credentials = await getCmrSettings();
```

Comment:
Nit, but avoiding the extra call and implicit test of `getCmrSettings` seems ideal

```suggestion
  const credentials = {};
```

Reply:
Ah, nice catch! I thought I got all of them.